migrations/7_deploy_abs_mock.js

```diff
@@ -0,0 +1,6 @@
+var ABSMock = artifacts.require("ABSMock")
```

Comment:
Does this need a migration? We could maybe migrate on the spot during the test, right?